### PR TITLE
Bellmanford update#150 Solved

### PR DESCRIPTION
**Pull request by jstjyoti**
@sahilbansal17

### CODE REVIEW

**Comment by sahilbansal17** [comment]
@jstjyoti This is fine but can you see the amount of commits that are being shown here? You should have made just one commit for the latest file after updating your master branch from upstream.

**Comment by jstjyoti** [comment]
yes, in hurry I repeated the mistake.Sorry

**Comment by sahilbansal17** [comment]
Its fine this time, take care from next time.

**Comment by jstjyoti** [comment]
yes, I will make sure. Another Issue I created and solved but that is closed. Should I add that solution?

**Comment by jstjyoti** [comment]
Kindly Review this once You get time.

**Comment by sahilbansal17** [comment]
@jstjyoti You can add the other solution but that won't fetch you points since you should have made sure that your issue was approved.

**Comment by jstjyoti** [comment]
Review this

**Comment by jstjyoti** [comment]
@sahilbansal17 @ramitsawhney27 Kindly review this problem was assigned to me and I have Solved this 1 day ago

**Comment by jstjyoti** [comment]
Okay

File: Algorithms/Graphs/Bellmanford.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<limits.h>/*for assigning maximum integer for all distances of vertices except for source vertex in function BellmanFord()*/
+//Name- Jyoti Mishra(@jstjyoti)
+//E mail-jyotimishra29091997@gmail.com
+/*Bellmanford's algorithm is a dynamic programming algorithm which can give all possible shortest part from a given source vertex to all the vertex in the graph containing negative or positive weighted edges and not having any negative weight cycle(a cycle of graph resulting negative weight as whole).It is a dynamic programming algorithm as need to use previous solutions of subproblems becuase we don't know future paths at the begining */
```
Comment (sahilbansal17) [inline Algorithms/Graphs/Bellmanford.c]: Then what does the **Warshall's algorithm** do?

Comment (sahilbansal17) [reply Algorithms/Graphs/Bellmanford.c]: Also, It is not clear how is it a Dynamic Programming algorithm?

Comment (jstjyoti) [reply Algorithms/Graphs/Bellmanford.c]: Okay I am Explaining in comment section.

**Comment by jstjyoti** [comment]
@sahilbansal17 I have Explained How it is different from FloydWarshall's Algorithm and how it is an DP method.
Kindly Review.

**Pull request merged**